app/assets/stylesheets/components/_supplier-table.scss

```diff
@@ -11,11 +13,7 @@
     .supplier-table__guarantee,
     .supplier-table__response,
     .supplier-table__complaints {
-      display: none;
-
-      @include cads-media-breakpoint-up(md) {
         display: table-cell;
```

Comment:
Minor: since there is no longer hiding of the cells depending on the breakpoint, the `display: table-cell` can be removed as it's the default display for table headings.